### PR TITLE
Maja Project Mongo api

### PR DESCRIPTION
**Pull request by majazimnoch**
https://project-mongo-api-4kmtou65hq-lz.a.run.app/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
 # Project Mongo API
 
-Replace this readme with your own information about your project.
+I started off with project by installing MangoDB and brew on my computer. 
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I added my own json file to the database generated by AI. The json fills consists of different web errors messages.
```
Comment (cecavi) [inline README.md:5]: Nice and descriptive README

File: data/errors.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
+[
+    {
```
Comment (cecavi) [inline data/errors.json:2]: I really like this database. Fun with errors. Did not know we hade this as an option in the example json files. Or maybe you found it somewhere else. Cool anyway.

File: server.js [hunk 3]
```diff
@@ -19,17 +20,112 @@ mongoose.Promise = Promise;
...
+    res.status(404).json({
+      success: false,
+      body: {
+        message: e
+      }
+    })
+  }
+});
+
+// Single endpoint. Error.countDocuments() method to get the total count of error documents in the database. Next, generate a random index within that range using Math.random() and Math.floor(). After that, Error.findOne().skip(randomIndex) method to retrieve a random error document by skipping to the randomly generated index.
```
Comment (cecavi) [inline server.js:107]: So thankful for your comments. I saw the code and was intrigued but did not understand it fully. Thanks to your comments I now understand it more.

**Review by cecavi — COMMENTED** [review]
Nice code and I see you got the database deployed. Good job!

File: data/errors.json (hunk 2 above)
Comment (majazimnoch) [reply data/errors.json:2]: I asked ChatGpt to write such json format for me :)

Comment (majazimnoch) [reply data/errors.json:2]: and then I just made a new file json and copied the text.

**Pull request closed without merging**